lumigator/python/mzai/backend/backend/api/routes/experiments.py

```diff
@@ -60,3 +70,54 @@ def get_experiment_result_download(
     return ExperimentResultDownloadResponse.model_validate(
         service.get_job_result_download(experiment_id).model_dump()
     )
+
+
+####################################################################################################
+# "new" routes
+####################################################################################################
+# These "experiments_new" routes are not yet ready to be exposed in the OpenAPI schema,
+# because it is designed for the API when 'workflows' are supported
+# TODO: Eventually this route will become the / route,
+# but right now it is a placeholder while we build up the Workflows routes
+# It's not included in the OpenAPI schema for now so it's not visible in the docs
+@router.post("/new", status_code=status.HTTP_201_CREATED, include_in_schema=False)
+def create_experiment_id(
+    service: ExperimentServiceDep, request: ExperimentIdCreate
+) -> ExperimentResponse:
+    """Create an experiment ID."""
+    return service.create_experiment(request)
+
+
+@router.get("/new/all", include_in_schema=False)
+def list_experiments_new(
+    service: ExperimentServiceDep,
+    skip: int = 0,
+    limit: int = 100,
+) -> ListingResponse[ExperimentResponse]:
+    """List all experiments."""
+    return ListingResponse[ExperimentResponse].model_validate(
+        service.list_experiments(skip, limit).model_dump()
+    )
+
+
+@router.get("/new/{experiment_id}", include_in_schema=False)
+def get_experiment_new(service: ExperimentServiceDep, experiment_id: UUID) -> ExperimentResponse:
+    """Get an experiment by ID."""
+    return ExperimentResponse.model_validate(service.get_experiment(experiment_id).model_dump())
+
+
+@router.get("/new/{experiment_id}/jobs", include_in_schema=False)
+def get_experiment_jobs(
+    service: ExperimentServiceDep, experiment_id: UUID
+) -> ListingResponse[UUID]:
+    """Get all jobs associated with an experiment."""
+    return service.get_all_owned_jobs(experiment_id)
```

Comment:
the method could be something a bit clearer like `get_all_jobs_by_experiment`

Reply:
Totally. I copied the naming from the way the function was currently named in the codebase, so I'll keep it this way for this PR but will update in a future PR 👍